.github/workflows/llk-auto-uplift-trigger.yaml

```
@@ -0,0 +1,1045 @@
+          SHOULD_RUN_WORMHOLE=false
+          SHOULD_RUN_BLACKHOLE=false
+
+          if echo "$CHANGED_FILES" | grep -q "^tt_llk_wormhole_b0/"; then
+            SHOULD_RUN_WORMHOLE=true
+            echo "✅ Detected Wormhole changes"
+          fi
+
+          if echo "$CHANGED_FILES" | grep -q "^tt_llk_blackhole/"; then
+            SHOULD_RUN_BLACKHOLE=true
+            echo "✅ Detected Blackhole changes"
+          fi
+
+          echo "should-run-wormhole=$SHOULD_RUN_WORMHOLE" >> $GITHUB_OUTPUT
+          echo "should-run-blackhole=$SHOULD_RUN_BLACKHOLE" >> $GITHUB_OUTPUT
+
+          # Generate commits table efficiently
+          echo "Found $COMMIT_COUNT commits using ${OLD_SHA}..${NEW_SHA}"
+
+          COMMITS_TABLE_CONTENT="| Commit | Message | Author | PR |"$'\n'
+          COMMITS_TABLE_CONTENT+="|--------|---------|---------|-----|"$'\n'
+
+          if [ "$COMMIT_COUNT" -gt 0 ]; then
+            git log --pretty=format:"%h|%s|%an|%H" ${OLD_SHA}..${NEW_SHA} | while IFS='|' read -r short_hash message author full_hash; do
+              pr_number=$(echo "$message" | grep -oE '\(#[0-9]+\)' | grep -oE '[0-9]+' | head -1)
+              clean_message=$(echo "$message" | sed 's/ *(#[0-9]\+) *$//')
+
+              if [ -n "$pr_number" ]; then
+                echo "| [$short_hash](https://github.com/tenstorrent/tt-llk/commit/$full_hash) | $clean_message | $author | [#$pr_number](https://github.com/tenstorrent/tt-llk/pull/$pr_number) |"
+              else
+                echo "| [$short_hash](https://github.com/tenstorrent/tt-llk/commit/$full_hash) | $clean_message | $author | - |"
+              fi
+            done >> /tmp/commits_table.txt
+
+            COMMITS_TABLE_CONTENT+=$(cat /tmp/commits_table.txt)
+          else
+            COMMITS_TABLE_CONTENT+="| (No commits found) | - | - | - |"$'\n'
+          fi
+
+          echo "commits-table<<EOF" >> $GITHUB_OUTPUT
+          echo "$COMMITS_TABLE_CONTENT" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+
+          # Generate initial PR body
+          cat > ../../../llk_commit_log.txt << EOF
+          ## ⬆️ LLK Submodule Update
+
+          Updating LLK submodule from \`$OLD_SHA\` to \`$NEW_SHA\`.
+
+          *This PR body will be updated with detailed information shortly...*
+          EOF
+
+      - name: Create Pull Request
+        id: create-pr
+        if: steps.update-submodule.outputs.has-changes == 'true'
+        uses: peter-evans/create-pull-request@v7
+        with:
+          token: ${{ secrets.TEMP_METAL_PAT }}
+          commit-message: "chore: update LLK submodule to ${{ steps.update-submodule.outputs.new-sha }}"
+          title: "chore: update LLK submodule to ${{ steps.update-submodule.outputs.new-sha }}"
+          branch: ${{ env.BRANCH_NAME }}
+          base: main
+          delete-branch: true
+          body-path: llk_commit_log.txt
+          add-paths: |
+            ${{ env.SUBMODULE_PATH }}
+          draft: ${{ !inputs.skip_draft }}
```

When this is triggered via cron, this will collapse into nothingness.  The Action may or may not be happy with that.  You'll probably want to try it out.  If it's not happy you can do something like this:

```suggestion
          draft: ${{ !inputs.skip_draft || true }} # or false -- however you want the auto-triggered ones to behave.
```

I'm not clear why/when we would want draft vs non-draft.  Why not just always do X and keep it simple/consistent?